tests/test_views.py

```diff
@@ -492,3 +495,38 @@ def test_refresh_jwt_after_refresh_expiration(self):
     def tearDown(self):
         # Restore original settings
         api_settings.JWT_ALLOW_REFRESH = DEFAULTS['JWT_ALLOW_REFRESH']
+
+
+class DeviceViewTests(TokenTestCase):
```

Comment:
Please add devices for not your user to make sure we are not showing it and can't delete.